resources/views/partials/bootstrap-table.blade.php

```diff
@@ -175,7 +175,9 @@ function hardwareAuditFormatter(value, row) {
 
 
     // Make the edit/delete buttons
-    function genericActionsFormatter(owner_name, element_name = '') {
+    function genericActionsFormatter(owner_name, element_name) {
+        if(!element_name) element_name = ''
```

Comment:
Another dumb couple of stylistic nits - microscopic - We try not do do one-line if's without the `{` and `}`. I think the reasoning is that you might naively break out the 'then' half into a second line and then might add a third line and think it's in the block - but it's not. Better to be explicit.

And finally, we like `;` on every 'statement' line, just because it helps us when we're switching back-and-forth between PHP and JS.

Reply:
Not dumb thanks for the insight, I need to start putting attention to this things.